Tasks/Common/vstsUtils.ts

```diff
@@ -16,6 +17,23 @@ export interface VSTSTaskManifest {
     version: VSTSManifestVersionInfo
 }
 
+export function warnIfBuildAgentTooLow(): boolean {
+    const version = getVariable('agent.version') ?? '0.0.0'
```

Comment:
also add a warning if the agent version is missing?

Reply:
That is possible, but it would require such an extremely old version